example/cumulus-tf/terraform.tfvars.example

```diff
@@ -126,3 +126,11 @@ additional_log_groups_to_elk = {
   "LogDescriptor" = "log_group_name"
 }
 */
+
+# Optional, configure if needed. Adds policies to SNS report topics.
+#  If a Principal is specified as just an AWS account ID rather than an ARN,
+#    AWS silently converts it to the ARN for the root user,
+#    causing future terraform plans to differ. To avoid this problem,
+#    specify the full ARN.
+# add_report_topic_policy = true
```

Comment:
Is it necessary to make it configurable?  sns topic has a policy (default or from terraform)

Reply:
@jennyhliu Yeah the topic will have a default policy, that's true. I added a configuration option because the ticket asked for it https://bugs.earthdata.nasa.gov/browse/CUMULUS-3377 
> We think it would be helpful for cumulus to add a flag for metrics integrations that when set to true, will update the access policy on the sns topics to match what we have in our documentation

But maybe since it is required for them to configure their metrics integration, maybe we make it required? Not sure 😅

Reply:
I meant we will create the policy anyway, w or w/o external subscribers

Reply:
Oh I see, yes the policy is created even without external subscribers. If you look at a topic's access policy that doesn't have these changes, you will notice it gets a default policy like:

```
{
  "Version": "2008-10-17",
  "Id": "__default_policy_ID",
  "Statement": [
    {
      "Sid": "__default_statement_ID",
      "Effect": "Allow",
      "Principal": {
        "AWS": "*"
      },
      "Action": [
        "SNS:GetTopicAttributes",
        "SNS:SetTopicAttributes",
        "SNS:AddPermission",
        "SNS:RemovePermission",
        "SNS:DeleteTopic",
        "SNS:Subscribe",
        "SNS:ListSubscriptionsByTopic",
        "SNS:Publish"
      ],
      "Resource": "arn:aws:sns:us-east-1:redacted:redacted-report-pdrs-topic",
      "Condition": {
        "StringEquals": {
          "AWS:SourceOwner": "redacted"
        }
      }
    }
  ]
}
```